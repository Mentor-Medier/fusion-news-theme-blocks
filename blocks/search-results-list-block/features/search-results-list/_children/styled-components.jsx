import styled from 'styled-components';

const HeadlineText = styled.h2`
  font-family: ${(props) => props.primaryFont};
`;

const DescriptionText = styled.p`
  font-family: ${(props) => props.secondaryFont};
`;

export { HeadlineText, DescriptionText };
