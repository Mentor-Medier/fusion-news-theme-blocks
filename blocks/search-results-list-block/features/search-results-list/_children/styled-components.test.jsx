// eslint-disable-next-line max-classes-per-file
import React from 'react';
import renderer from 'react-test-renderer';
import { HeadlineText, DescriptionText } from './styled-components';


describe('the HeadlineText', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(<HeadlineText>Text</HeadlineText>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});

describe('the DescriptionText', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(<DescriptionText>Description</DescriptionText>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
