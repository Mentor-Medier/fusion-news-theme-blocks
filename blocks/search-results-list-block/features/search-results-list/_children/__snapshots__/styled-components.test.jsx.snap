// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`the DescriptionText renders correctly 1`] = `
<p
  className="sc-bwzfXH gfyHkX"
>
  Description
</p>
`;

exports[`the HeadlineText renders correctly 1`] = `
<h2
  className="sc-bdVaJa gDGmfb"
>
  Text
</h2>
`;
